test/stability/testMerge/merge_test.go

```diff
@@ -8,9 +8,30 @@ import (
 	"testing"
 )
 
+func TestInit(t *testing.T) {
+	if err := initDPDK(); err != nil {
+		t.Fatalf("fail to initialize with error: %+v\n", err)
+	}
+}
+
+// TODO unfortunately this test can't be executed as go test right now due to https://github.com/intel-go/nff-go/issues/301
+// If you want to test it via go test you should firstly comment FreeMempools function at flow/flow.go
+// You will have multiple mempools however test will pass.
 func TestMerge(t *testing.T) {
-	if err := executeTest("", "", 0); err != nil {
-		t.Logf("fail: %+v\n", err)
-		t.Fail()
+	scenarios := []pipelineScenario{getGet, inCopy, copyCopy, segmSegm, segmGet, segmCopy, inSegm, manyFlows}
+	gens := []getScenario{fastGenerate}
```

Comment:
Do we want to check generate case too?
If test become too long, can we run some tests with generate and some with fastGenerate?

Reply:
We can't use generate with copy scenarios, cause speed is too large and test fails due to drops.
But with scenarios without copy it can be done. @ifilippov, do we really need it?

Reply:
We don't want to have generate here. It will be part of InputOutput tests. Fast generate is enough, you can remove slice and use only fast Generate everywhere.